api.go

```diff
@@ -27,22 +27,22 @@ import (
 func CreatePod(podConfig PodConfig) (*Pod, error) {
 	// Create the network.
 	n := newNetwork(podConfig.NetworkModel)
-	networkNS, err := n.add(&(podConfig.NetworkConfig))
+	networkNS, err := n.add(Pod{}, &(podConfig.NetworkConfig))
```

Comment:
@mcastelino yes I agree the add() function creates some data and we want to know them from the caller, that's why I pass the NetworkConfig pointer. The pod is used here only as a value on purpose (it wouldn't hurt to have a pointer though). Regarding QEMU parameters, take a look at add() implementation, it calls an hypervisor (qemu in our case) API called AddDevice() to add the appropriate network configuration to qemu parameters.